<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>351</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>751</width>
    <height>351</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="frm_dynamics" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>737</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f0f0f0</string>
   </property>
   <widget class="QWidget" name="slider_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>500</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>341</height>
     </size>
    </property>
    <widget class="QSlider" name="sld_x_0">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>22</width>
       <height>281</height>
      </rect>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSlider" name="sld_y_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>22</width>
       <height>281</height>
      </rect>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSlider" name="sld_z_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>22</width>
       <height>281</height>
      </rect>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSlider" name="sld_roll_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>22</width>
       <height>281</height>
      </rect>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSlider" name="sld_pitch_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>22</width>
       <height>281</height>
      </rect>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSlider" name="sld_yaw_5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>22</width>
       <height>281</height>
      </rect>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>310</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>133</x>
       <y>310</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>213</x>
       <y>310</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sld_roll">
     <property name="geometry">
      <rect>
       <x>283</x>
       <y>310</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Roll</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sld_pitch">
     <property name="geometry">
      <rect>
       <x>364</x>
       <y>310</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pitch</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sld_yaw">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Yaw</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="wgt_overlay" native="true">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>20</y>
       <width>16</width>
       <height>95</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0,0,20)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>106</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QWidget" name="wgt_overlay_2" native="true">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>20</y>
       <width>16</width>
       <height>95</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0,0,20)</string>
     </property>
    </widget>
    <widget class="QWidget" name="wgt_overlay_3" native="true">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>20</y>
       <width>16</width>
       <height>95</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0,0,20)</string>
     </property>
    </widget>
    <widget class="QWidget" name="wgt_overlay_4" native="true">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>20</y>
       <width>16</width>
       <height>95</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0,0,20)</string>
     </property>
    </widget>
    <widget class="QWidget" name="wgt_overlay_6" native="true">
     <property name="geometry">
      <rect>
       <x>364</x>
       <y>20</y>
       <width>16</width>
       <height>95</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0,0,20)</string>
     </property>
    </widget>
    <widget class="QWidget" name="wgt_overlay_7" native="true">
     <property name="geometry">
      <rect>
       <x>444</x>
       <y>20</y>
       <width>16</width>
       <height>95</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0,0,20)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="master_widget" native="true">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>0</y>
      <width>120</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>351</height>
     </size>
    </property>
    <widget class="QLabel" name="lbl_master">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Master</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="sld_master_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>22</width>
       <height>281</height>
      </rect>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QWidget" name="wgt_overlay_5" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>16</width>
       <height>95</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0,0,20)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="button_widget" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>101</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>351</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_reload_dynam">
       <property name="text">
        <string>Reload</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_default_dynam">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_save_dynam">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
